source/billing.rst

```diff
@@ -2,14 +2,22 @@
 Billing service
 ###############
 
-There are many different ways to get your billing details using our system.
-We offer a billing API, that is easy to access and consume using a python
+There are many different ways to get your billing details using our system. The usage
+costs report shows you an estimate for your usage within the current region.
+
+It provides a breakdown per service, so you can quickly spot which services you are using
+the most and where your savings opportunities are. It also shows you the aggregate cost
+per month over the last twelve months, so you can identify usage patterns and project the
+expected usage for the current month.
+
+Our billing API provides an easy way to access and consume using a python
```

Comment:
This sentence doesn't sound right. (genuinely don't know but) does the API provide a way to access and consume the data or does the script access and consume data from the API?